packages/app/src/app/pages/Sandbox/Editor/index.tsx

```diff
@@ -19,6 +19,9 @@ import { Navigation as NavigationOld } from './NavigationOld';
 import { ContentSkeleton } from './Skeleton';
 import getVSCodeTheme from './utils/get-vscode-theme';
 import { Workspace } from './Workspace';
+import { CommentsGlobalStyles } from './Workspace/screens/Comments/components/GlobalStyles';
+import { MultiComment } from './Workspace/screens/Comments/components/MultiComment';
+import { CommentDialog } from './Workspace/screens/Comments/Dialog';
```

Comment:
totally optional enhancement: I would wrap these 3 in the same file /comments/index and treat it as the _external api_ for comments while all of the other components are internal implementation detail

Reply:
The issue is that we already have an index.tsx that is the actual entry point for comments

maybe an API.tsx would make sense and its a good middle ground?

Reply:
oooh, i like it!

Reply:
Done! :D